modules/ocf_mirrors/files/project/kde-appdata/sync-archive

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -eu
+/usr/local/bin/rsync-no-vanished -az --delete --safe-links \
+	rsync.kde.org::applicationdata /opt/mirrors/ftp/kde-applicationdata
```

Comment:
Make sure to also remove this from `modules/ocf_mirrors/files/project/kde/sync-archive`